docs/SetupDatabaseForAzure.md

```diff
@@ -1,72 +1,199 @@
 
-* You should configure the backup policy as `Continuous` for Point-in-time-restore (PITR).
-* You should configure the service endpoint based on the [Azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/how-to-configure-vnet-service-endpoint) to restrict access to Cosmos DB for production.
+* [Getting Started with ScalarDB on Cosmos DB](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started-with-scalardb-on-cosmosdb.md)
 
-### Steps
+### Required configuration/steps
 
-* Create an Azure Cosmos DB account based on the [Azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/create-cosmosdb-resources-portal#create-an-azure-cosmos-db-account) with the above requirements.
-* (Optional) Configure advanced monitoring services with [Monitor Azure Cosmos DB](https://docs.microsoft.com/en-us/azure/cosmos-db/monitor-cosmos-db) guide, by default monitoring (metrics) will be enabled in Cosmos DB.
-* (Optional) Update the `cosmosBaseResourceUnit` value in the [schema-loading-custom-values](https://github.com/scalar-labs/scalar-kubernetes/blob/master/conf/schema-loading-custom-values.yaml) to scale the throughput of Cosmos DB.
+#### Create an Azure Cosmos DB account
 
-Note:-
+You must create an Azure Cosmos DB account with the NoSQL (core) API. You must set the **Capacity mode** as **Provisioned throughput** when you create it. Please refer to the official document for more details.
 
-* The [scalardl-schema-loader](https://github.com/scalar-labs/scalardl-schema-loader) applies the `cosmosBaseResourceUnit` value (the default value is 10) to all tables.
-* The scalardl-schema-loader enables autoscale of [Request Units](https://docs.microsoft.com/en-us/azure/cosmos-db/request-units) (RU) for all tables.
+* [Quickstart: Create an Azure Cosmos DB account, database, container, and items from the Azure portal](https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/quickstart-portal)
 
+#### Configure a default consistency configuration
+
+You must set the **Default consistency level** as **Strong**. Please refer to the official document for more details.
+
+* [Configure the default consistency level](https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/how-to-manage-consistency#config/ure-the-default-consistency-level)
+
+### Optional configurations/steps
+
+#### Configure backup configurations (Recommended in the production environment)
+
+You can configure **Backup modes** as **Continuous backup mode** for PITR. Please refer to the official document for more details.
+
+* [Backup modes](https://learn.microsoft.com/en-us/azure/cosmos-db/online-backup-and-restore#backup-modes)
+
+It is recommended since the continuous backup mode automatically and continuously gets backups so that we can reduce the downtime (pause duration) of backup operations. Please refer to the following document for more details on how to backup/restore the Scalar product data.
```

Comment:
```suggestion
It is recommended since the continuous backup mode automatically and continuously gets backups so that we can reduce downtime (pause duration) for backup operations. Please refer to the following document for more details on how to backup/restore the Scalar product data.
```